packages/atomic/src/components/search/atomic-breadbox/atomic-breadbox.mdx

```diff
@@ -0,0 +1,45 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+import * as AtomicBreadboxStories from './atomic-breadbox.new.stories';
+import { AtomicDocTemplate } from '../../../../storybook-utils/documentation/atomic-doc-template';
+
+<Meta of={AtomicBreadboxStories} />
+
+<AtomicDocTemplate
+  stories={AtomicBreadboxStories}
+  githubPath="search/atomic-breadbox/atomic-breadbox.ts"
+  tagName="atomic-breadbox"
+  className="AtomicBreadbox"
+>
+
+The `atomic-breadbox` component creates breadcrumbs that display a summary of the currently active facet values.
```

Comment:
This 100% duplicates the component's JSDoc, which will be rendered right above :D 

We can remove that.